weaviate/collections/classes/aggregate.py

```diff
@@ -497,24 +576,24 @@ def date_(
         )
 
     # Aggregate references currently bugged on Weaviate's side
-    # def reference(
-    #     self,
-    #     pointing_to: bool = False,
-    # ) -> _MetricsReference:
-    #     """Define the metrics to be returned for a cross-reference property when aggregating over a collection.
-
-    #     If none of the arguments are provided then all metrics will be returned.
-
-    #     Arguments:
-    #         `pointing_to`
-    #             Whether to include the collection names that this property references.
-
-    #     Returns:
-    #         A `_MetricsReference` object that includes the metrics to be returned.
-    #     """
-    #     if not any([pointing_to]):
-    #         pointing_to = True
-    #     return _MetricsReference(
-    #         property_name=self.__property,
-    #         pointing_to=pointing_to,
-    #     )
+    def reference(
+        self,
+        pointing_to: bool = False,
+    ) -> _MetricsReference:
+        """Define the metrics to be returned for a cross-reference property when aggregating over a collection.
+
+        If none of the arguments are provided then all metrics will be returned.
+
+        Arguments:
+            `pointing_to`
+                Whether to include the collection names that this property references.
```

Comment:
IIRC it is the objects it is pointing to, eg it aggregates over the reference property to see which objects are referenced how often.

Reply:
With that new context, the migration tooling will be vastly improved with aggregating xrefs now allowed again

Reply:
better double check then. I think I did this almost 2 years ago and don't remember all details